### ISSUE TITLE
Implement the Footer Section

### ISSUE THREAD

**Issue by orama254** [reactdeveloperske/reactdevske-website#83]
### Depends on

- Logo Component #84 

### What feature are you adding?

Footer Section

### Requirements

- Create a component named `Footer` in the `components` folder that implements the screenshots below.
- The menu items are anchor links which should smoothly scroll to the relevant section when clicked.
- The sections do not need to exist for this ticket, only ensure that the links are correct.
- Reuse the `Logo` component.
- Include the component in `index.page.tsx`.
- Create a new test file in the `e2e` folder named `footer.spec.ts` and add a test that verifies that clicking each of the 3 links in the component navigates to the correct URLs.

**SCREENSHOTS**

**Desktop**
![footer-section](https://user-images.githubusercontent.com/50571688/194324342-d4eb1994-ba4f-4b14-a85b-3c9c939869d7.png)

**Mobile**
![footer-section-mobile](https://user-images.githubusercontent.com/50571688/194679037-5780db9e-6d2c-4ed0-b575-3b7c1315fb7e.png)

[Figma Design](https://www.figma.com/file/TVwnaDhBGeVdnVKdf6H91C/React-developers-community-website?node-id=1321%3A1626)

### Acceptance Criteria

- [ ] When the "About Us" menu is clicked, the URL changes to `/#about-us`.
- [ ] When the "Events" menu is clicked, the URL changes to `/#events`.
- [ ] When the "Contact" menu is clicked, the URL changes to `/#contact-us`.
- [ ] The implementation should match the design - text content, typography, capitalization and spacing.
- [ ] The UI should be responsive and work well on both desktop and mobile viewports.
- [ ] The `footer.spec.ts` test should pass on both Chrome, Firefox and Safari.

**Comment by yashsarnaik22** [comment]
hey @orama254 can I do this?
please assign it to me.

**Comment by orama254** [comment]
Hey @yashsarnaik22 

Go ahead, remember to go through our [contribution guidelines](https://github.com/reactdeveloperske/reactdevske-website/blob/main/CONTRIBUTING.md) and also evaluate the acceptance criteria before submitting a pull request.

Also note that this issue depends on #84 . Check it out an let me know if you are ok implementing it or can wait for it to be implemented.

Happy hacking!!!

**Comment by orama254** [comment]
Yes, @yashsarnaik22 that is ok.

for quality purposes I would advise that you make a separate PR for issue #84 first before creating a PR for this issue.

**Comment by yashsarnaik22** [comment]
Thank you! On it.

**Comment by yashsarnaik22** [comment]
@orama254 the header section is not done yet? I wanted to provide href  to the links

**Comment by antosan** [comment]
> @orama254 the header section is not done yet? I wanted to provide href to the links

@yashsarnaik22 This ticket does not depend on the Header section. The links that should be provided are already described in the acceptance criteria above. They might be broken at first, but once all the other tickets are done, the relevant sections where they link will be present.

This ticket only depends on #84, which you have to wait to be implemented and merged since you need to reuse the logo in the footer.

**Comment by orama254** [comment]
Hey @yashsarnaik22 

I'm following up to see how far you have reached with the implementation of this issue.

Let me know if you also need assistance on anything.

**Comment by orama254** [comment]
Hello @devKiratu would you be up for this task?

**Comment by devKiratu** [comment]
> Hello @devKiratu would you be up for this task?

Hi @orama254 , Sure! Please assign me
